.github/workflows/submodule_update.yml

```diff
@@ -4,50 +4,94 @@ on:
   push:
     branches:
       - 'master'
-      - '2.10'
+      - '[2-9].\d+'
+
+env:
+  EE_FEATURE_BRANCH: 'TarantoolBot/update-tarantool-${{ github.ref_name }}'
 
 jobs:
   tarantool-ee:
     # Run on push to the 'master' and release branches in tarantool/tarantool
     # repo only.
     if: github.repository == 'tarantool/tarantool'
 
-    runs-on: [self-hosted, Linux, x86_64, flavor-1-2]
+    runs-on: [ self-hosted, Linux, x86_64, flavor-1-2 ]
```

Comment:
It should be fixed in the previous commit.

Reply:
oops, you're right